extension-readme.md

```diff
@@ -1,134 +1,97 @@
-1. Create and configure a set of AWS credentials (see [Establishing Credentials](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/establish-credentials.html) in the user guide)
-1. Connect the Toolkit to AWS using those credentials (see [Connecting to AWS](https://docs.aws.amazon.com/console/toolkit-for-vscode/connect) in the user guide)
+{ [Return to Top](#top) }
 
-To use the Toolkit to develop [serverless applications with AWS](https://aws.amazon.com/serverless/), you must also [set up your toolchain](https://docs.aws.amazon.com/console/toolkit-for-vscode/setup-toolchain) and do the following on the local machine where the Toolkit is installed:
+## <a id="ui-components-cdk-expl"></a>CDK Explorer
 
-1. Install the AWS Command Line Interface (AWS CLI)
-1. Install and start Docker (also see this general information about [Docker](https://docs.docker.com/install/))
-1. Install the AWS SAM CLI
+The **AWS CDK Explorer** enables you to work with AWS Cloud Development Kit (CDK) applications. Its tree view shows a top-level view of your CDK applications, infrastructure stacks, resources, and policies.
 
-For complete setup instructions for these three components, see [Installing the AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) in the _AWS Serverless Application Model (AWS SAM) Developer Guide_.
+![Overview, AWS CDK Explorer](./resources/marketplace/overview-cdk.png)
 
-{ [Return to Contents](#contents) } or { [Return to Top](#top) }
+{ [Return to Top](#top) }
 
 ---
 
-# <a id="features"></a>Features
-
--   [Serverless Applications, Lambda Functions, and CloudFormation Stacks](#sam-and-lambda)
--   [AWS CDK Applications](#cdk-apps)
--   [Amazon EventBridge Schemas](#eventbridge)
--   [Amazon ECS Task Definition Files](#ecs-files)
--   [AWS Step Functions](#sfn-files)
+## <a id="sam-and-lambda"></a> AWS Serverless Applications
 
-{ [Return to Contents](#contents) } or { [Return to Top](#top) }
+The AWS Toolkit enables you to develop [AWS serverless applications](https://aws.amazon.com/serverless/) locally. It also provides _CodeLenses_ in VS Code to do the following:
 
----
+-   Use SAM (serverless application model) templates to build and debug your locally developed AWS serverless applications.
+-   Run selected [AWS Lambda](https://aws.amazon.com/lambda/) functions.
 
-## <a id="sam-and-lambda"></a>Serverless Applications, Lambda Functions, and CloudFormation Stacks
+After you've [created a serverless application](https://docs.aws.amazon.com/console/toolkit-for-vscode/create-sam), you can locally run and debug it using a SAM template. This YAML-formatted file describes resources such as functions, APIs, databases, and event-source mappings that make up a serverless application.
```

Comment:
This paragraph is pretty much the same as the one two lines above, but does not read as well